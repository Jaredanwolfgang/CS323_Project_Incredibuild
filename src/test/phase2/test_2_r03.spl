struct Student
{
    int weight;
    float weight;
};
int test_2_r03()
{
    struct Student amy;
    int age;
    float age;
    return 0;
}
