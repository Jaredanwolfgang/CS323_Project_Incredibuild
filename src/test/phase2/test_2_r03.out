[Semantic Error] Redefinition Error[15] at line 3: "weight" redefined.
[Semantic Error] Undefined Reference Error[14] at line 8: "Student" undefined.
[Semantic Error] Redefinition Error[3] at line 10: "age" redefined.